README.md

```diff
@@ -169,8 +169,8 @@ This can be handy for testing your account out, or if you want to override your
 
 You can run `nancy` in a quiet manner, only getting back a list of vulnerable components by running:
 
-* `./nancy -quiet /path/to/your/Gopkg.lock `
-* `./nancy -quiet /path/to/your/go.sum `
+* `./nancy -quiet /path/to/your/Gopkg.lock`
```

Comment:
Pretty sure it's `-p` now for path yeah?

Reply:
> Pretty sure it's `-p` now for path yeah?

Good catch! I also swapped things to document default==quiet, and new --loud flag.  cfdd7de